.github/workflows/reusable_docs_build.yml

```diff
@@ -21,11 +21,13 @@ jobs:
         sudo apt-get update
         sudo apt-get install -y doxygen
 
-    - name: Install pip requirements
-      run: python3 -m pip install -r third_party/requirements.txt
-
-    - name: Setup PATH for python
-      run: echo "$HOME/.local/bin" >> $GITHUB_PATH
```

Comment:
Isn't it needed any more?

Reply:
nope, it's part of the `Install Python requirements in venv` step, pls see: `echo "$PATH" >> $GITHUB_PATH` (prev. it was installed in this `.local/bin` dir, now it's installed within venv)